src/gened/instructor.py

```diff
@@ -85,6 +86,31 @@ def _get_class_users(*, for_export: bool = False) -> list[Row]:
 
     return users
 
+def _get_query_limits_data() -> list[Row]:
+    cur_class = get_auth_class()
+    class_id = cur_class.class_id
+    db = get_db()
+
+    users = db.execute("""
+        SELECT
+            users.id,
+            json_array(users.display_name, auth_providers.name, users.display_extra) AS user,
+            users.queries_used,
+            classes.max_queries,
+            ('<form method="POST" action="/instructor/reset_student_queries/' || users.id || 
```

Comment:
Generating HTML in the SQL statement is an overly-complex way to accomplish this.  I think using the `actions` list in the DataTable can handle this in a much simpler way.  Let me know if that doesn't work here for some reason.